### PR TITLE
Adding share price to wallet info

### PR DESCRIPTION
**Pull request by slundqui**

### CODE REVIEW

File: elfpy/data/acquire_data.py [hunk 1]
```diff
@@ -10,12 +10,15 @@
...
 from web3.contract.contract import Contract
 
-from elfpy.data import postgres
 from elfpy import eth, hyperdrive_interface
+from elfpy.data import postgres
 from elfpy.utils import logs as log_utils
 
 # pylint: disable=too-many-arguments
 
+# TODO fix too many branches by splitting out various things into functions
```
Comment (dpaiton) [inline elfpy/data/acquire_data.py:19]: can you make an issue to split up acquire_data in general? same with how I split up contract_interfaces.

File: elfpy/hyperdrive_interface/hyperdrive_interface.py [hunk 2]
```diff
@@ -203,6 +204,8 @@ def get_wallet_info(
         token_maturity_time = transaction.event_maturity_time
         if wallet_addr is None:
             continue
+
```
Comment (dpaiton) [inline elfpy/hyperdrive_interface/hyperdrive_interface.py:207]: no newlines pls

**Review by dpaiton — APPROVED** [review]

File: elfpy/hyperdrive_interface/hyperdrive_interface.py (hunk 2 above)
Comment (slundqui) [reply elfpy/hyperdrive_interface/hyperdrive_interface.py:207]: This is breaking up logic and I'm using it sparingly

Comment (slundqui) [reply elfpy/hyperdrive_interface/hyperdrive_interface.py:207]: I'll remove for now, but I'll fight you tooth and nail on this. For me, breaking up logic with new lines makes the code much easier to read (comments doesn't do the same for me)

**Pull request merged**